e2e/test/jest-restart-after-runners/stryker.conf.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../../node_modules/@stryker-mutator/core/schema/stryker-schema.json",
+  "packageManager": "npm",
+  "testRunner": "jest",
+  "tempDirName": "stryker-tmp",
+  "concurrency": 1,
+  "restartAfterRuns": 1,
```

Comment:
I set it to 1 for this e2e test because this e2e only has 3 mutants, and I wanted to show it restarting *multiple* times. Is there a different approach you'd prefer?

Reply:
What I mean is that this e2e test doesn't _actually need the test runner process to restart after each run_. I've tested it out by setting `restartAfterRuns` to `0` and the e2e test still passed. 

I think it would be better if we added a test here that fails the second time it is run.

I also find this test pretty expensive. In order to test the decorator, we're adding an e2e test package that requires it's `dependencies` and `localDependencies`. This means adding time it takes to `install` the e2e tests. That's fine if you're testing something specific use case that has its own maintenance lifecycle (i.e. angular cli, vue cli, react scripts, etc) , but since we're just testing the decorator, it's pretty expensive.

All that being said, maybe an end to end test is not the right level. We have a whole badge of integration tests in `packages/core/test/integration/test-runner`, maybe it's better to add the tests for the `maxTestRunnerReuse` decorator there?